test/headers_test.py

```diff
@@ -0,0 +1,41 @@
+import niquests
+import uuid
+from unittest.mock import patch, Mock
+from seam.client import SeamHttpClient
+from importlib.metadata import version
+from seam.constants import LTS_VERSION
+
+
+def test_seam_http_client_request(server):
+    endpoint, seed = server
+    client = SeamHttpClient(
```

Comment:
Is is possible to test it this way?

```py
seam = Seam(...)
client = seam.client
```

Reply:
Sure! d7d2406945c00d8ad8cc48ba0d9e6e00f9b5e0f4